onnx/onnx-ml.proto

```diff
@@ -252,7 +252,7 @@ message GraphProto {
 
   // A list of named tensor values, used to specify constant inputs of the graph.
   // Each TensorProto entry must have a distinct name (within the list) that
-  // also appears in the input list.
+  // MAY also appear in the input list.
```

Comment:
We should also tell users which are recommended to appear in the input list, for example, some weights may be overwritten. Some constants won't be changed for sure, they should not be included in the initializers.

Reply:
I added this explanation to IR.md, since that seemed like a good place for this.

Reply:
@houseroad I am in the middle of updating the IR_VERSION. It looks like I may have to regenerate the python notebooks in the onnx\examples folder since I am running into a failure there during testing. Please let me know how these files are generated or how to update them.